learn/dynamic-application-configuration/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/hashicorp/waypoint-examples/learn/dynamic-application-configuration
+
+go 1.17
```

Comment:
Should we try this with 1.19 before it's merged?

Reply:
I'll give it a whirl!